### ISSUE TITLE
508-defect-4 [COGNITION]: Help text should describe the field instead of being a part of the label

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#19616]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Josh_

## Details

[Hint text would be better provided following the guidance in this slack thread](https://dsva.slack.com/archives/C0NGDDXME/p1612301886124500?thread_ts=1612298930.119600&cid=C0NGDDXME) where it describes the label as opposed to being a part of it. This will steer us towards more consistent usage of hint text in fields.

## Acceptance Criteria

- [ ] Hint text follows future design system and accessibility specialist guidance

### Before
<img width="1226" alt="Screen Shot 2021-02-08 at 4 10 54 PM" src="https://user-images.githubusercontent.com/14154792/107281928-57404d00-6a28-11eb-8d6d-016f53aca82e.png">

### After
<img width="1465" alt="Screen Shot 2021-02-08 at 4 09 29 PM" src="https://user-images.githubusercontent.com/14154792/107281911-51e30280-6a28-11eb-95b0-f7585456b1ee.png">

**Comment by mshea0606** [comment]
@1Copenut is this a form system issue? cc @erikphansen @sporeboy

**Comment by 1Copenut** [comment]
@mshea0606 Yes, it reads as a design system issue. We have a ticket started here: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/396

**Comment by mshea0606** [comment]
ok thanks @1Copenut!  cc/ @erikphansen @sporeboy

**Comment by joshkimux** [comment]
[This is blocked by #1334](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1334)

**Comment by humancompanion-usds** [comment]
I believe this issue is now superseded by https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1786
